LiskHQ/lisk-desktop
# Stake state is not cleaned when changing account

### What was the problem?

This PR resolves #5442

### How was it solved?

Always clean stakes when switching account, application or network. 

### How was it tested?

1. Stake 10 LSK to validator1 from account1
2. Switch to an account which has not staked to validator1
3. Navigate to validator1
4. Expected: Not show that edit stake for that validator
